azurerm/helpers/validate/cosmos_test.go

```diff
@@ -132,10 +132,6 @@ func TestCosmosMaxThroughput(t *testing.T) {
 			Value:  1000000,
 			Errors: 0,
 		},
-		{
-			Value:  1100000,
-			Errors: 1,
-		},
```

Comment:
Personally I would leave this test in the test suite but change Errors to expect 0 to verify that the upper limit on throughput has been removed. WDYT?